pages/landing/patcher-use-cases/index.html

```diff
@@ -0,0 +1,87 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Patcher Use Cases
+excerpt: Some examples of scenarios where Patcher can be useful
+permalink: /patcher-use-cases/
+custom_js:
+- prism
+prism_css: /assets/css/prism-landing-zone.css
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Request early access!
+  logos:
+  - /assets/img/logos-technologies/partner_logo_aws.png
+  - /assets/img/logos-technologies/partner_logo_kubernetes.png
+  - /assets/img/logos-technologies/partner_logo_terraform.png
+
+use_cases:
+  - heading: "Use case 1: Update to Terraform 1.0 automatically"
+    tab_block: site.data.patcher-landing-page-tabs-terraform-upgrade
+    body: |
+      You can use Patcher to automatically update a codebase containing Terraform modules to a new Terraform version, including new major versions that contain backwards incompatible changes. Patcher will help you update your codebase all the way from Terrafrom 0.11 to Terraform 1.0, and it supports incremental upgrades  (e.g., 0.11 -> 0.12 -> 0.13 -> 0.14 -> 0.15 -> 1.0). Patcher will scan your codebase and open a pull request, such as the one shown in the screenshot below, that transforms your code to use the new version of Terraform. For example, Terraform 0.12 had a particularly painful backwards incompatible upgrade, but Patcher can make such upgrades as painless and smooth as reviewing and merging a PR.
+      At Gruntwork, we have created patches to automatically do each of the incremental upgrades. E.g. the patch that preforms the upgrade from 0.13 to 0.14 supports making these changes to the codebase: update `required_version` constraint in code; replace version constraints in `provider` blocks with `required_providers` configuration; update all the versions in `source` URLs to use the versions of Gruntwork modules that work with TF 0.14; update CI / CD config to use TF 0.14; update Terratest version to the version that supports TF 0.14; run `terraform fmt`; run `terraform validate`.
```

Comment:
Not sure how to format text as lists/code inside the `body` block.

Reply:
Also, this is too much text for this one area. Having a list will help make it more readable, but it may still be too long. Here's my attempt to tighten things up a bit:

![Screen Shot 2021-08-11 at 7 30 32 PM](https://user-images.githubusercontent.com/711908/129083730-1a46105c-e929-49a0-a67a-658ea16defb2.png)

Note that I added some CSS styles for `<code>...</code>`: `background: #6d6d6d; color: #e0e0e0`.

Reply:
Thanks for these pointers! Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/583/commits/acf7dc112afef421d034cef849c84ef908c87efc.